docs/source/examples/tabular/checks/integrity/source/plot_outlier_sample_detection.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+"""
+Outlier Sample Detection
+***************
+
+This notebooks provides an overview for using and understanding the Outlier Sample Detection check.
+
+**Structure:**
+
+* `How deepchecks detects outliers <#How-deepchecks-detects-outliers>`__
+* `Prepare data <#prepare-data>`__
+* `Run the check <#run-the-check>`__
+* `Define a condition <#define-a-condition>`__
+
+How deepchecks detects outliers
+========================
+Outlier Sample Detection searches for outliers samples using the LoOP algorithm.
```

Comment:
Maybe first a word on the fact that the outliers are multivariate - considering all features at once.